tools/notification-configuration/common/common.csproj

```diff
@@ -1,10 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Kusto.Data" Version="7.1.0" />
```

Comment:
Why does this need to be moved to the common?

Reply:
So that I can reference it from a common location.